common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -152,6 +157,13 @@ public BrokerAcquireTokenOperationParameters brokerInteractiveParametersFromActi
 
         parameters.setAuthorizationAgent(AuthorizationAgent.WEBVIEW);
 
+        // Set Global environment variable for instance discovery if present
```

Comment:
Do we have to add this in getAcquireTokenParametersForBrokerRTRequest()? (or somehow refactor the code in a  way that we don't have to dupe this logic, if that's the case).

Reply:
We would have set the global parameter by then, i.e. when the client first calls acquire token we set the environment given by the client